frontend/src/pages/Assignments/Assignment/index.tsx

```diff
@@ -148,22 +187,8 @@ const Assignment: React.FC = () => {
           <div className="Assignment__subSectionWrapper">
             <h2 style={{ marginBottom: 10 }}>Metrics</h2>
             <MetricPanel>
-              <SimpleMetric
-                metricTitle="First Commit Date"
-                metricValue="6 Sep"
-              ></SimpleMetric>
-              <SimpleMetric
-                metricTitle="Total Commits"
-                metricValue="941"
-              ></SimpleMetric>
-              <SimpleMetric
-                metricTitle="Extension  Requests"
-                metricValue="0"
-              ></SimpleMetric>
-              <SimpleMetric
-                metricTitle="Regrade  Requests"
-                metricValue="0"
-              ></SimpleMetric>
+              <SimpleMetric metricTitle="First Commit Date" metricValue={firstCommit}></SimpleMetric>
```

Comment:
Should this be on the assignment page? Is this a valuable metric?

Reply:
idk man